src/wp-admin/includes/class-wp-plugins-list-table.php

```diff
@@ -584,6 +582,25 @@ protected function get_views() {
 						$count
 					);
 					break;
+				default:
+					/**
+					 * Filter the status text of default switch case in the Plugins list table.
+					 *
+					 * @since 6.9.0
+					 *
+					 * @param string $text   Plugins list status text. Default empty string.
+					 * @param int    $count  Count the Number of plugins.
+					 * @param string $type   The status slug being filtered.
+					 */
+					$text = apply_filters( 'plugins_list_status_text', '', $count, $type );
+					if ( empty( $text ) ) {
+						$text = $type;
+					}
+					$text .= ' ' . sprintf(
```

Comment:
Maybe not strictly necessary but for sanity's sake:

```suggestion
					$text = esc_html( $text ) . ' ' . sprintf(
```